modBoot/resource/genesis.json

```diff
@@ -1,7 +1,13 @@
 {
   "alloc": {
-    "0xa0eeaeabdbc92953b072afbd21f3e3fd8a4a4f5e6a6e22200db746ab75e9a99a": {
+    "0xa048613dd3cb89685cb3f9cfa410ecf606c7ec7320e721edacd194050828c6b0": {
```

Comment:
genesis file change should not be merged in

Reply:
This is my bad, It will be fixed